### PR TITLE
Clarify Comments and Fix Typos

### PR DESCRIPTION
**Pull request by bmurray7**
The existing comment is unclear and has grammatical errors.

### CODE REVIEW

**Comment by bmurray7** [comment]
I've aggregated several minor typo fixes into this PR.  Please let me know if would prefer me to create separate PRs for each change.

File: library/net_sockets.c [hunk 1]
```diff
@@ -238,7 +238,7 @@ int mbedtls_net_bind( mbedtls_net_context *ctx, const char *bind_ip, const char
             }
         }
 
-        /* I we ever get there, it's a success */
+        /* Bind was sucsessfull  */
```
Comment (sbutcher-arm) [inline library/net_sockets.c]: I hate to point this out, but you've misspelt successful.

**Comment by sbutcher-arm** [comment]
LGTM

**Comment by sbutcher-arm** [comment]
Reviewed again, and approved again, following the recent additional commits and still ready for merge.

@bmurray7 - I don't normally add to the ChangeLog a record for typos, but you've fixed so many, I feel I have to say thanks again! You have read a lot of our code and comments to get this many.

**Pull request merged**